### PR TITLE
Advancing 0.13.4 release to status: projects

### PR DESCRIPTION
**Pull request by dfarrell07**
Advancing 0.13.4 release to status: projects

### CODE REVIEW

File: releases/v0.13.4.yaml [hunk 1]
```diff
@@ -2,7 +2,10 @@
 version: v0.13.4
 name: 0.13.4
 branch: release-0.13
-status: admiral
+status: projects
 components:
   shipyard: 32d409523c0f5ac0316650048e410284d0f1e2d5
   admiral: a43edec490db6d1ed7c658b9d8424095fde92d04
+  cloud-prepare: e6a38ef07ef71c571463627eaf58b9246aaba5f5
```
Comment (dfarrell07) [inline releases/v0.13.4.yaml:9]: https://github.com/submariner-io/cloud-prepare/commit/e6a38ef07ef71c571463627eaf58b9246aaba5f5

File: releases/v0.13.4.yaml [hunk 2]
```diff
@@ -2,7 +2,10 @@
...
+  lighthouse: df100fb3eacfc23f8377ab2969ec66cd0f00872b
```
Comment (dfarrell07) [inline releases/v0.13.4.yaml:10]: https://github.com/submariner-io/lighthouse/commit/df100fb3eacfc23f8377ab2969ec66cd0f00872b

File: releases/v0.13.4.yaml [hunk 3]
```diff
@@ -2,7 +2,10 @@
...
+  submariner: 7c6078e874fcd4da132be06fc46e0c01448d4382
```
Comment (dfarrell07) [inline releases/v0.13.4.yaml:11]: https://github.com/submariner-io/submariner/commit/7c6078e874fcd4da132be06fc46e0c01448d4382

**Review by tpantelis — APPROVED** [review]

**Comment by dfarrell07** [comment]
CC @aswinsuryan

**Review by aswinsuryan — APPROVED** [review]

**Pull request merged**